google-datacatalog-sisense-connector/setup.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import setuptools
+
+release_status = 'Development Status :: 4 - Beta'
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+setuptools.setup(
+    name='google-datacatalog-sisense-connector',
+    version='0.1.0',
```

Comment:
Any reason to start at `0.1.0`?

Reply:
Nice catch. I changed to `0.0.1` and will bump to `0.1.0` before publishing the first working version.